Solutions/Armis/Playbooks/ArmisUpdateAlertStatus/azuredeploy.json

```diff
@@ -0,0 +1,543 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
+        "title":  "Armis Update Alert Status",
+        "description":  "Armis Update Alert Status playbook would be responsible to update the Alert status from the sentinel to the Armis Portal",
+        "prerequisites":  [
+            "1. The Armis Alerts data connector should be configured to send appropriate armis alerts events to Microsoft Sentinel.",
+            "2. Store Armis API secret key in Key Vault and obtain keyvault name and tenantId.",
+                "a. Create a Key Vault with unique name",
+                "b. Go to KeyVault -> secrets -> Generate/import and create 'ArmisAPISecretKey' for storing Armis API Secret Key"
+        ],
+        "postDeployment":  [
+            "**a. Authorize connections**",
+            "Once deployment is complete, authorize each connection.",
+            "1. Click the Microsoft Sentinel connection resource",
+            "2. Click edit API connection",
+            "3. Click Authorize",
+            "4. Sign in",
+            "5. Click Save",
+            "6. Repeat steps for other connections",
+            "**b. Configurations in Sentinel**",
+            "1. In Microsoft Sentinel, analytical rules should be configured to trigger an incident. An incident should have the *alertID* - custom entity that contains alertId of each generated Armis alert and *alertStatus* - custom entity that contains alertStatus of each generated Armis alerts. It can be obtained from the corresponding field in Armis Alerts custom logs. Check the [documentation](https://docs.microsoft.com/azure/sentinel/surface-custom-details-in-alerts) to learn more about adding custom entities to incidents.",
+            "2. Configure the automation rules to trigger the playbook."
+        ],
+        "prerequisitesDeployTemplateFile":  "",
```

Comment:
Please remove "prerequisitesDeployTemplateFile" if not required or used, thanks.